ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v1/container/clientlibs/editor/js/editDialog.js

```diff
@@ -367,6 +367,8 @@
             restEndPointUrlTextBox = Utils.selectElement("input", './restEndpointPostUrl')[0],
             restEndpointConfigPath = Utils.selectElement("input", './restEndpointConfigPath')[0];
 
+        if (!enableRestEndpointCheckboxElement) return;
```

Comment:
Why do you need this check, does not seem relevant for this use-case

Reply:
the javascript breaks when the rest endpoint url is not selected from the submit action options, hence, this was required.

Reply:
Updated the code.